subsys/nrf_rpc/include/nrf_rpc_log.h

```diff
@@ -15,7 +15,17 @@
 #define _NRF_RPC_LOG_REGISTER1(module) \
 	_NRF_RPC_LOG_REGISTER2(module)
 
+#define _NRF_RPC_LOG_DECLARE2(module) \
```

Comment:
Why we need those two levels of macro ?

Reply:
If I were to use only level 2, then "CONFIG_ ## module ## _LOG_LEVEL", would result in appending name of module macro, not its value. Level 1 makes sure to pass only value of level 1s define.